sdk/quantum/azure-quantum-jobs/src/main/java/module-info.java

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.quantum.jobs {
+    exports com.azure.quantum.jobs;
+
+    requires com.azure.core;
+    requires com.azure.identity;
+
+    exports com.azure.quantum.jobs.models to com.fasterxml.jackson.databind;
```

Comment:
This line can be removed since this is already open to jackson databind.

Reply:
Done!